shards/tests/hello.shs

```diff
@@ -200,3 +200,9 @@ WhenNot(@foo-predicate(-1) {Msg("More than -2")})
 "Hello World!" | String.Starts(With: "Hello") | Assert.Is(true) | Log
 ; also test that doesn't start with
 "Hello World!" | LogType("Type logging test") | String.Starts(With: "World") | Assert.Is(false) | Log
+
+["1" "2" "3"] = zs1
+["4" "5" "6"] = zs2
+["7" "8" "9"] = zs3
+["10" "20" "30" "40"] = zs4
+[zs1 zs2 zs3 zs4] | Zip | Assert.Is([["1" "4" "7" "10"] ["2" "5" "8" "20"] ["3" "6" "9" "30"]]) | Log("Zipped")
```

Comment:
Maybe better as Zip(a b c d ...), can pass by reference in that case too

Reply:
Have to do some manual setParam magic

Reply:
But we don't have N params hmm

Reply:
Never mind, we better just use a single seq param and allow vars & const sequences inside into a single parameter